src/qmozview_p.cpp

```diff
@@ -835,6 +835,10 @@ void QMozViewPrivate::setMargins(const QMargins &margins, bool updateTopBottom)
 
 void QMozViewPrivate::OnLocationChanged(const char *aLocation, bool aCanGoBack, bool aCanGoForward)
 {
+    if (QString(aLocation) == QStringLiteral("about:blank") && !aCanGoBack && !aCanGoForward) {
```

Comment:
```suggestion
    if (QLatin1String(aLocation) == QLatin1String("about:blank") && !aCanGoBack && !aCanGoForward) {
```
We don't really need QString instances. Not that big of a deal.